clients/githubrepo/graphql.go

```diff
@@ -86,8 +86,16 @@ type graphqlData struct {
 		Issues struct {
 			Nodes []struct {
 				// nolint: revive,stylecheck // naming according to githubv4 convention.
-				Url       *string
-				UpdatedAt *time.Time
+				Url               *string
+				AuthorAssociation githubv4.String
+				CreatedAt         githubv4.DateTime
+				ClosedAt          githubv4.DateTime
+				Comments          struct {
+					Nodes []struct {
+						AuthorAssociation githubv4.String
+						CreatedAt         githubv4.DateTime
+					}
```

Comment:
Using pointer values here will easily let you know when the API returns no data. E.g `AuthorAssociation` would be `nil` instead of "" when an association could not be made.